Pyrado/scripts/plotting/plot_policy_domain_grid_1d.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2020, Fabio Muratore, Honda Research Institute Europe GmbH, and
```

Comment:
please separate the addition of the new file from the actual content of this PR

Reply:
whoopsy

Reply:
Done with 275179c.